components/esm/spellstate.cpp

```diff
@@ -33,23 +33,36 @@ namespace ESM
             mSpells[id] = state;
         }
 
+        // Obsolete
```

Comment:
This is confusing. Reader may have impression that this code can be removed. If we don't plan to remove save compatibility this code is not obsolete. I hope we don't. I'd better to rewrite it in the obvious way that it just discards the result with a comment that this should not be removed until we keep compatibility with saves from <= 0.46.

Reply:
I just took the approach which we use in many other places. There is a lot of such "obsolete", "unused" and "deprecated" stuff, and it is not really corprus-specific.

> I hope we don't. 

I suppose we eventually will have to, and it is better to do it prior to 1.0 release.